src/service/calendar.ts

```diff
@@ -0,0 +1,69 @@
+// External Dependencies
+import * as axios from 'axios'
+
+// Client
+import ClubHubClient from '../client'
+
+// Local Namespace
+import Calendar from 'src/models/calendar'
+import Event from 'src/models/event'
+
+/**
+ * Interface to the ClubHub `Calendar` API.
+ */
+export default class CalendarService {
+
+  /**
+   * The `ClubHubClient` instance for the service. 
+   */
+  public client: ClubHubClient
+
+  constructor(client: ClubHubClient) {
+    this.client = client
```

Comment:
Looks like you have different indentation for spacing. Can we go with tab indentation for this project